src/WebApiContrib.Testing/ApiControllerExtensions.cs

```diff
@@ -39,6 +39,7 @@ public static void ConfigureForTesting(this ApiController controller, HttpReques
             controller.Request = request;
             controller.Request.Properties[HttpPropertyKeys.HttpConfigurationKey] = config;
             controller.Request.Properties[HttpPropertyKeys.HttpRouteDataKey] = routeData;
+            controller.RequestContext = new HttpRequestContext { Configuration = config, RouteData = routeData, Url = new UrlHelper(controller.Request) };
```

Comment:
This is the only thing that really changed. The rest is dependency version and conversion to `async`/`await` from the `TaskHelper`.